template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/ranger/RangerCloudStorageServiceConfigProvider.java

```diff
@@ -18,13 +21,28 @@
 @Component
 public class RangerCloudStorageServiceConfigProvider implements CmTemplateComponentConfigProvider {
 
-    private static final String RANGER_HDFS_AUDIT_URL = "ranger_plugin_hdfs_audit_url";
+    static final String RANGER_HDFS_AUDIT_URL = "ranger_plugin_hdfs_audit_url";
+
+    static final String RANGER_DATA_URL = "cloud_data_location_url";
+
+    static final String RANGER_LOG_URL = "cloud_log_location_url";
 
     @Override
     public List<ApiClusterTemplateConfig> getServiceConfigs(CmTemplateProcessor templateProcessor, TemplatePreparationObject templatePreparationObject) {
-        return ConfigUtils.getStorageLocationForServiceProperty(templatePreparationObject, RANGER_HDFS_AUDIT_URL)
-                .map(location -> List.of(config(RANGER_HDFS_AUDIT_URL, location.getValue())))
-                .orElseGet(() -> List.of(config(RANGER_HDFS_AUDIT_URL, getDefaultRangerAuditUrl(templateProcessor, templatePreparationObject))));
+        List<ApiClusterTemplateConfig> rangerConfigs = new ArrayList<>();
+
+        ConfigUtils.getStorageLocationForServiceProperty(templatePreparationObject, RANGER_HDFS_AUDIT_URL)
```

Comment:
nit: This whole thing is hard to parse. Consider rewriting with extracting some of the variables.
```
        List<ApiClusterTemplateConfig> rangerConfigs = new ArrayList<>();

        final String rangerHdfsAuditUrl = ConfigUtils.getStorageLocationForServiceProperty(templatePreparationObject, RANGER_HDFS_AUDIT_URL)
            .map(location -> location.getValue())
            .orElseGet(() -> getDefaultRangerAuditUrl(templateProcessor, templatePreparationObject));

       rangerConfigs.add(config(RANGER_HDFS_AUDIT_URL, rangerHdfsAuditUrl));

       final String rangerDataUrl = ...
       final String rangerLogUrl = ...
       rangerConfigs.add(...);
       rangerConfigs.add(...);

      return rangerConfigs;
```